Isaac Lab Setup (#4)
khristova21/revolution: setup/README.md

# Installation

> TODO: Add FFMPEG insatllation to the script, so we can record videos of the agents.

Reviewer: ```suggestion
TODO: Add FFMPEG installation to the script, so we can record videos of the agents.
```

Simply run the setup script on Windows or Linux for a streamlined install. Note: 50-series Nvidia cards [require a nightly build of PyTorch](https://developer.nvidia.com/cuda-gpus) as of 5/13/2025. This is incorporated into the script.

# Common Issues

This is how to debug common issues that comes with the Isaac Lab installation process. The setup scripts attempts to streamline the process, but there may be a few kinks. Feel free to follow the official [installation guide](https://isaac-sim.github.io/IsaacLab/main/source/setup/quickstart.html#quick-installation-guide) for custom installation (for instance, using conda or miniconda versus our script which uses pip).

## Linux & Windows Issues

### Black Viewport

Activate the virtual enviroment `isaac_env` and run `isaacsim` to verify the simulator runs as expected.  

> Ensure the simulation is ran on your discrete graphics processing unit, and verify you meet the [minimum requirements](https://docs.isaacsim.omniverse.nvidia.com/4.5.0/installation/requirements.html). Use the Isaac Sim Compatibility Checker if you are unsure.  

Reviewer: ```suggestion
Ensure the simulation has run on your discrete graphics processing unit, and verify you meet the [minimum requirements](https://docs.isaacsim.omniverse.nvidia.com/4.5.0/installation/requirements.html). Use the Isaac Sim Compatibility Checker if you are unsure.  
```
